Attachment_Downloader/attachment.py

```diff
@@ -0,0 +1,44 @@
+import ezgmail
```

Comment:
How will new users authenticate?

Reply:
Please follow setup instructions section of readme file. Credential.json and token.json files are generated once user is authenticated.